README.md

```diff
@@ -33,15 +34,24 @@ is distributed under the
 
 # Contents
 
+- [Saturn — Parallelism-Safe Data Structures for Multicore OCaml](#saturn--parallelism-safe-data-structures-for-multicore-ocaml)
```

Comment:
This is not an issue with this PR, but some time ago GitHub started to automatically produce a kind contents or navigation drop down for .md files.  So, I've been thinking of potentially starting to drop contents section from .md files (less stuff to maintain).  On quick look ocaml.org/packages doesn't yet produce such a navigation helper, so I guess there is still value in having a contents section.  So, just mention this here as a potential thing to discuss.